src/client/containers/Home/Home.js

```diff
@@ -42,20 +42,25 @@ export const Home = () => {
         </div>
         <div className="home-button">
           <div className="home-button-right">
-            <Buttons
-              label="Take the career quiz"
-              size="big"
-              isMono={true}
-              color="orange"
-            />
+            <Link to={'/CareerPage'}>
+              <Buttons
+                label="Take the career quiz"
+                size="big"
+                isMono={true}
+                color="orange"
```

Comment:
I think, we do not need to pass the color-orange props as the default is mentioned as orange :)